gnd/src/main/java/com/google/android/gnd/ui/map/gms/GoogleMapsMapAdapter.java

```diff
@@ -277,28 +302,33 @@ private void addMapPin(MapPin mapPin) {
   }
 
   private void addMapPolyline(MapPolygon mapPolygon) {
-    for (ImmutableSet<Point> vertices : mapPolygon.getVertices()) {
-      PolylineOptions options = new PolylineOptions();
-
-      // Read-only
-      options.clickable(false);
-
-      // Add vertices to PolylineOptions
-      stream(vertices).map(GoogleMapsMapAdapter::toLatLng).forEach(options::add);
-
-      // Add to map
-      Polyline polyline = map.addPolyline(options);
-      polyline.setTag(mapPolygon);
+    PolylineOptions options = new PolylineOptions();
+    options.clickable(false);
+    ImmutableList<LatLng> vertices = stream(mapPolygon.getVertices())
+        .map(GoogleMapsMapAdapter::toLatLng)
+        .collect(toImmutableList());
+    options.addAll(vertices);
+    // Add to map
```

Comment:
"Add to map" just repeats what's already on the next line ("map.addPolyline"); please only use comments when it's not possible to make your code self-explanatory. From Robert C. Martin's Clean Code:

> The proper use of comments is to compensate for our failure to express ourself in code. Note that I used the word failure. I meant it. Comments are always failures. We must have them because we cannot always figure out how to express ourselves without them, but their use is not a cause for celebration.... Every time you express yourself in code, you should pat yourself on the back. Every time you write a comment, you should grimace and feel the failure of your ability of expression.